demo/run-tmux.py

```diff
@@ -0,0 +1,168 @@
+import libtmux
+import os
+import subprocess
+import sys
+import time
+
+
+if not os.path.exists('demo'):
+    raise Exception(
+        'Plase navigate to the root directory of the hydra-poc project'
+    )
+
+os.chdir('demo')
+
+server = libtmux.Server()
+
+SESSION_NAME = 'hydra-demo'
+session = server.find_where({'session_name': SESSION_NAME})
+
+
+def send_cmd(pane, cmd):
+    pane.send_keys(cmd)
+    time.sleep(1)
+    pane.enter()
+
+
+if not session:
+    session = server.new_session(session_name=SESSION_NAME)
```

Comment:
This is failing for me if I have no tmux running. Only if there is another session somewhere this works.